docs/cas-server-documentation/protocol/REST-Protocol.md

```diff
@@ -138,6 +138,42 @@ the generated identifier of the new service.
 5463544213
 ```
 
+## Add x509 support
+
+The feature extends REST communication model to non-interactive authentication.
+This pattern may be of interest in case if internal network architecture hides
+the CAS server from external users behind firewall or some messaging bus. And
+allows only trusted Applications to connect the server.
+
+<div class="alert alert-warning"><strong>Usage Warning!</strong><p>The x509 feature over REST
+provides a tremendously convenient target for claiming others identity. The x509 provides
+a tremendously convenient target for claiming others identity. To leverage this issue network
+configuration may allow connections to the CAS server only from trusted hosts which in turn
+have strict security limitations and logging.
```

Comment:
I have lost </p></div> for warning block. I have already fixed the bug in my clone repo. Should I create one more PR?